client/state/selectors/is-iframe-for-html-element.js

```diff
@@ -1,6 +0,0 @@
-import getSectionName from 'calypso/state/ui/selectors/get-section-name';
-import getSelectedSiteId from 'calypso/state/ui/selectors/get-selected-site-id';
-
-const isIframeForHtmlElement = ( state ) =>
-	!! ( getSelectedSiteId( state ) && getSectionName( state ) === 'gutenberg-editor' );
```

Comment:
This is the other half of the cleanup - we can no longer be in this section since we're no longer using the iframed editor.